internal/consensus/pbts_test.go

```diff
@@ -31,6 +31,9 @@ type pbtsTestHarness struct {
 	// configuration options set by the user of the test harness.
 	pbtsTestConfiguration
 
+	// The timestamp of the first block produced by the network.
+	firstHeight time.Time
```

Comment:
IMO the `Time` suffix was useful: `x.firstHeight` reads like a height (vs., say, `x.firstHeightTime` or `x.firstBlockTime` or whatever)